src/App/Pages/Settings/AutofillServicesPageViewModel.cs

```diff
@@ -192,7 +193,8 @@ public void ToggleDrawOver()
         
         public void UpdateEnabled()
         {
-            AutofillServiceToggled = _deviceActionService.AutofillServiceEnabled();
+            AutofillServiceToggled =
+                Device.RuntimePlatform == Device.Android && _deviceActionService.AutofillServiceEnabled();
```

Comment:
Even though this is fine, I think the `_deviceActionService` should be the one indicating whether `AutofillServiceEnabled()` should be called given that following the **Information expert** pattern that service is the one with the information to check whether that method is implemented and IMO we should minimize the runtime platform checks on the VMs if possible.
So something like this would be better
`AutofillServiceToggled = _deviceActionService.HasAutofillService && _deviceActionService.AutofillServiceEnabled();`

Reply:
Makes sense - I went ahead and made the change and will save the remainder for a tech debt task (there's a lot)